### DISCUSSION TITLE
Noise suppression? When use is not wearing headphones

### DISCUSSION THREAD

**Discussion by wasauce** [deepgram/community#221] [General help] [answered]
Hello!

Does the Deepgram team or this community have suggestions for how we might improve our transcriptions in scenarios where a user is on . a Slack call, a Google Meet and our recording has their mic picking up sound (words) from the other person (i.e. the person they are talking to on a slack call)?

We see that the transcription ends up quite odd because the user is not wearing headphones. We try to tell users to use headphones but some simply refuse or forget.

Thank you!

**Comment by wasauce** [comment]
One thought is to use https://github.com/Rikorose/DeepFilterNet but I'm unsure how well this will work. Especially in a real time scenario.

**Reply by lukeocodes** [reply, marked as answer]
We don't apply noise suppression to the audio we train on, so that it can more accurately handle audio that has background noise.

But, noise suppression on your audio would lead to less false positives, picking up words from background noise.

You could try an opensource solution, like [noise-suppression-for-voice](https://github.com/werman/noise-suppression-for-voice), a free service like [utterly](https://www.utterly.app/), or something like krisp.ai

Let us know how you get on, or what solution you come up with!
